e2e/smartvol_ops_test.go

```diff
@@ -60,6 +59,18 @@ func testSmartVolumeDistribute(t *testing.T) {
 	r.Nil(brickSizeTest(volinfo.Subvols[1].Bricks[0].Path, 16, 21))
 	r.Nil(brickSizeTest(volinfo.Subvols[2].Bricks[0].Path, 16, 21))
 
+	expandReq := api.VolExpandReq{
+		Size:            60 * deviceutils.MiB,
+		DistributeCount: 3,
+	}
+	_, err = client.VolumeExpand(smartvolname, expandReq)
+	r.Nil(err)
+	vols, err := client.Volumes(smartvolname)
```

Comment:
new line, each test should be in separate block

Reply:
Done